### PR TITLE
Add tests on TestMetaTaskProps toString

### PR DESCRIPTION
**Pull request by feloy**
**What this PR does / why we need it**:

Test TestMetaTaskProps toString

**Which issue this PR fixes** *(optional, in `fixes #<issue number>(, fixes #<issue_number>, ...)` format, will close that issue when PR gets merged)*: fixes #

fixes https://github.com/openebs/openebs/issues/1585

**Special notes for your reviewer**:

### CODE REVIEW

**Comment by sonasingh46** [comment]
@feloy 
Please give issue link in PR description. (Not just the number)
e.g.
fixes <issue_link>

File: pkg/task/meta_test.go [hunk 1]
```diff
@@ -1526,3 +1526,40 @@ action: {{ .action }}
...
+			},
+			result: "runNamespace=aRunNamespace::owner=anOwner::objectName=anObjectName::options=someOptions::retry=someRetry",
+		},
+	}
+
+	for name, mock := range tests {
+		t.Run(name, func(t *testing.T) {
+			res := mock.metaTaskProps.toString()
+			if res != mock.result {
+				t.Fatalf("metaTaskProps string represntation should be '%s': actual result is '%s'", mock.result, res)
```
Comment (princerachit) [inline pkg/task/meta_test.go]: For sake of uniformity let us have similar message syntax. PTAL at above UTs to see message.

Comment (feloy) [reply pkg/task/meta_test.go]: I made some changes

**Comment by sonasingh46** [comment]
@feloy 
Thanks for the PR.
Closing this PR due to inactivity. Please let us know of we want to work on the same.

**Pull request closed without merging**